test/calculator_test.js

```diff
@@ -0,0 +1,44 @@
+const assert = require('assert');
+const chai = require('chai');
+const calc = require('../calculator')
+
+describe('calculator', function () {
+  describe('#calculate()', function () {
```

Comment:
Is there a reason why there's a hash here? Just wanted to double check that this was the standard formatting for this repo! 😃